packages/components/ovh-payment-method/src/ovh-payment-method.js

```diff
@@ -275,15 +205,12 @@ export const useOvhPaymentMethod = ({ reketInstance, region }) => {
    *  @return {Promise} That returns the default payment method instance.
    */
   const getDefaultPaymentMethod = () => {
-    return getAllPaymentMethods({
+    return getPaymentMethods({
       onlyValid: true,
       transform: true,
     }).then(
       (paymentMethods) =>
-        find(
-          paymentMethods,
-          (method) => method.default || method.defaultPaymentMean,
-        ) || null,
+        find(paymentMethods, (method) => method.default) || null,
```

Comment:
As you modified that part, you can remove `lodash` as well

```suggestion
        paymentMethods.find((method) => method.default) || null,
```